e2e/wdio/platform/tests/icon-tab-bar.e2e-spec.ts

```diff
@@ -0,0 +1,172 @@
+        });
+    });
+
+    describe('Icon only example', () => {
+        it('should check selecting tabs in iconName only example', () => {
+            checkSelectingTabs(iconOnlyExample);
+        });
+
+        // skipped due to https://github.com/SAP/fundamental-ngx/issues/6714
+        xit('should check that no labels in only iconName example', () => {
+            expect(doesItExist(iconOnlyExample + label)).toBe(false, 'label exists but should not');
+            expect(doesItExist(iconOnlyExample + counter)).toBe(false, 'counter exists but should not');
+        });
+    });
+
+    describe('Nested tabs example', () => {
+        it('should check selecting tabs in nested example', () => {
+            checkSelectingTabs(nestedTabsExample);
+        });
+
+        // skipped due to https://github.com/SAP/fundamental-ngx/issues/6743
+        xit('should check tab with expanded list', () => {
+            click(nestedTabsExample + tabBarItem, 3);
+            expect(isElementDisplayed(expandedList)).toBe(true, 'expanded list is not displayed');
+            const itemText = getText(listItem, 1);
+            click(listItem, 1);
+            expect(getText(nestedTabsExample + tabBarItem)).toEqual(itemText, 'text is not changed according to selected item');
+            expect(getAttributeByName(nestedTabsExample + tabBarTab, 'aria-selected', 3)).toBe('true', 'tab is not selected');
+        });
+
+    });
+
+    describe('Reordering example', () => {
+        it('should check selecting tabs in columns example', () => {
+            checkSelectingTabs(reorderingExample);
+        });
+    });
+
+    describe('Overflowing example', () => {
+        it('should check selecting tabs in overflowing example', () => {
+            refreshPage();
+            checkSelectingTabs(overflowingExample);
+        });
+
+        it('should check overflowing', () => {
+            click(overflowButton);
+            expect(isElementDisplayed(expandedList)).toBe(true, 'expanded list is not displayed');
+            const iconName = getElementClass(listItem + icon);
+            const quantityOfDisplayedTabs = getElementArrayLength(overflowingExample + tabBarItem + '[aria-hidden=false]');
+            click(listItem);
+            expect(getElementClass(overflowingExample + tabBarItem + icon, quantityOfDisplayedTabs - 1)).toBe(iconName, 'tabs are not replaced');
+        });
+
+    });
+
+    describe('Configurations paddings example', () => {
+        it('should check selecting tabs in configuratable paddings example', () => {
+            checkSelectingTabs(configuratablePaddingsExample);
+        });
+
+        it('should check paddings between tabs', () => {
+            for (let i = 0; i < 4; i++) {
+                expect(getElementClass(configuratablePaddingsExample + tabBar, i)).toContain(paddingsSizes[i], 'paddings are not of sm size');
+            }
```

Comment:
please update error message; not expecting all of them to be sm
`paddings are not ${paddingSizes[i]} size`